pages/contributors.js

```diff
@@ -0,0 +1,220 @@
+import axios from "axios"
+import React from "react"
+import Head from "next/head"
+import { useEffect, useState } from "react"
+import animationData from "../lottie/sad.json"
+import { Text } from "atomize"
+import { Spinner } from "react-bootstrap"
+import Lottie from "react-lottie"
+import Card from "../components/Cards/Card"
+
+const defaultOptions = {
+    loop: true,
+    autoplay: true,
+    animationData: animationData,
+}
+
+export default function Contributors() {
+    const [contriFrontend, setContributorsFront] = useState()
+    const [contriBackend, setContributorsBack] = useState()
+    const [contributors, setContributors] = useState()
+    const [error, setError] = useState([false, false, false])
```

Comment:
Why the 3rd element?

Reply:
It is the Final Array for the contributors

Reply:
No i'm talking about the 3rd element in the error array

Reply:
oh! yes, its not necessary, I'll remove it.
In fact only one is necessary.